app/server/src/features/accounts/methods.ts

```diff
@@ -19,43 +17,13 @@ const toUserId = toGlobalId('User');
 
 type MinimalUser = Pick<RegistrationForm, 'email'> & Partial<Pick<RegistrationForm, 'firstName' | 'lastName'>>;
 
-/**
- * given an email, returns if email exists on csv file on google cloud storage bucket
- */
-export async function isOnOrganizerList(email: string) {
-    // const bucketName = 'organizers';
-    // const fileName = 'list-of-organizers.csv';
-    // const storage = new Storage({
-    //     projectId: process.env.GCP_PROJECT_ID,
-    //     keyFilename: process.env.GOOGLE_APPLICATION_CREDENTIALS,
-    // });
-    // const file = storage.bucket(bucketName).file(fileName);
-
-    let found = false;
-    // const options = {
-    //     destination: process.env.ORGANIZER_FILE_PATH,
-    // };
-    // try {
-    //     await file.download(options);
-    // } catch (err) {
-    //     throw new Error(err);
-    // }
-
-    // not working rn for some reason
-    let localFilePath = __dirname + '/listOfOrganizers.csv'; //delete later
-    const allFileContents = fs.readFileSync(/*process.env.ORGANIZER_FILE_PATH*/ localFilePath, 'utf-8');
-    allFileContents.split(/\r?\n?,/).forEach((line: string) => {
-        if (line === email) {
-            found = true;
-        }
-    });
-    return found;
-    // return true;
-}
-
 export async function isOrganizer(userId: string, prisma: PrismaClient) {
     const queryResult = await prisma.user.findUnique({ where: { id: userId }, select: { canMakeOrgs: true } });
-    if (!queryResult) throw new Error(errors.DNE('user'));
+    if (!queryResult)
+        throw new ProtectedError({
+            userMessage: 'Internal server error. Please try again later.',
+            internalMessage: errors.DNE('user'),
```

Comment:
I think making the message more detailed makes sense here since it's an internal error. 

`User with id ${userId} is not allowed to make orgs or does not exist.`